examples/random/main.go

```diff
@@ -90,9 +90,10 @@ func main() {
 		for {
 			v := (rand.NormFloat64() * *normDomain) + *normMean
 			rpcDurations.WithLabelValues("normal").Observe(v)
-			rpcDurationsHistogram.ObserveWithExemplar(
-				// Demonstrate exemplar support with a dummy ID. This would be
-				// something like a trace ID in a real application.
+			// Demonstrate exemplar support with a dummy ID. This would be
+			// something like a trace ID in a real application.
+			// Note the necessary type assertion.
+			rpcDurationsHistogram.(prometheus.ExemplarObserver).ObserveWithExemplar(
```

Comment:
I would maybe in this example check if it's not implementing this? So:

```
if e, ok := rpcDurationsHistogram.(prometheus.ExemplarObserver); ok {
  e..ObserveWithExemplar( ...
}
```

Reply:
This is because encouraging avoiding panics in your code is strongly suggested... But one can argue that we directly call our library constructor, so up to you here, not a blocker (:

Reply:
Yeah, it's all in the same file. I have added a comment now that explains why we don't check the type assertion.

Reply:
And I'll also fix the "tipo". :o(

Reply:
Thanks!